apps/arbeidsforhold-api/src/test/java/no/nav/registre/testnorge/arbeidsforhold/provider/ArbeidsforholdControllerIntegrationTest.java

```diff
@@ -0,0 +1,127 @@
+    private ObjectMapper objectMapper;
+
+    private static final String orgnummer = "123456789";
+    private static final String personIdent = "12345678910";
+    private static final String arbeidsforholdId = "1";
+    private static final String yrke = "Tester";
+    private static final double stillingsprosent = 100.00;
+    private static final String aaregUrl = "(.*)/aareg-test/api/v1/arbeidstaker/arbeidsforhold";
+    private static final String tokenUrl = "(.*)/token-provider";
+
+    private no.nav.registre.testnorge.arbeidsforhold.consumer.dto.ArbeidsforholdDTO[] aaregResponse;
+
+    @Before
+    public void setUp() {
+        var arbeidsforhold = no.nav.registre.testnorge.arbeidsforhold.consumer.dto.ArbeidsforholdDTO.builder()
+                .arbeidsforholdId(arbeidsforholdId)
+                .arbeidsgiver(ArbeidsgiverDTO.builder().organisasjonsnummer(orgnummer).build())
+                .arbeidsavtaler(Collections.singletonList(ArbeidsavtaleDTO.builder()
+                        .stillingsprosent(stillingsprosent)
+                        .yrke(yrke)
+                        .build()))
+                .ansettelsesperiode(AnsettelsesperiodeDTO.builder()
+                        .periode(PeriodeDTO.builder()
+                                .fom(LocalDate.now().minusMonths(3))
+                                .tom(LocalDate.now())
+                                .build())
+                        .build())
+                .arbeidstaker(ArbeidstakerDTO.builder()
+                        .offentligIdent(personIdent)
+                        .build())
+                .build();
+
+        aaregResponse = new no.nav.registre.testnorge.arbeidsforhold.consumer.dto.ArbeidsforholdDTO[] { arbeidsforhold };
+    }
+
+    @Test
+    public void shouldGetArbeidsforhold() throws Exception {
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(tokenUrl)
+                .stubGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(aaregUrl)
+                .withResponseBody(aaregResponse)
+                .stubGet();
+
+        var mvcResultat = mockMvc.perform(get("/api/v1/arbeidsforhold/" + personIdent)
+                .contentType(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk())
+                .andReturn()
+                .getResponse()
+                .getContentAsString();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(tokenUrl)
+                .verifyGet();
+
+        JsonWiremockHelper
+                .builder(objectMapper)
+                .withUrlPathMatching(aaregUrl)
+                .withResponseBody(Collections.singletonList(aaregResponse))
+                .verifyGet();
+
+        var resultat = objectMapper.readValue(mvcResultat,
+                ArbeidsforholdDTO[].class);
+
+        assertThat(resultat).hasSize(1);
```

Comment:
Vill prøed å lage et nytt opbject for å sammenligne:

` assertThat(resultat).hasSize(1).contains(new Arbideforhold(1, 12344, ....))`

Reply:
God ide, har fikset :)